packages/firestore/src/core/firestore_client.ts

```diff
@@ -70,6 +71,17 @@ const DOM_EXCEPTION_INVALID_STATE = 11;
 const DOM_EXCEPTION_ABORTED = 20;
 const DOM_EXCEPTION_QUOTA_EXCEEDED = 22;
 
+/**
+ * Initial backoff time in milliseconds after an error.
+ * Set to 1s according to https://cloud.google.com/apis/design/errors.
+ */
+const BACKOFF_INITIAL_DELAY_MS = 1000;
+
+/** Maximum backoff time in milliseconds */
+const BACKOFF_MAX_DELAY_MS = 60 * 1000;
+
+const BACKOFF_FACTOR = 1.5;
```

Comment:
These constants look really familiar!

Maybe we should move them to backoff.ts and just make them be default values for the ExponentialBackoff() constructor?